--- Makefile.in.dist	2005-02-09 16:27:48.406257536 +0100
+++ Makefile.in	2005-02-09 16:35:54.626340848 +0100
@@ -5,7 +5,7 @@
 libexecdir=@libexecdir@
 doxygen=@doxygen@
 bindir=$(prefix)/bin
-mandir=$(prefix)/man
+mandir=@mandir@
 INSTALL=@INSTALL@
 
 SRCS=nbsmtp.c base64.c util.c smtp.c fdutil.c original.c fileconfig.c @EXTRASRCS@ main.c
@@ -28,12 +28,12 @@
 	$(RM) config.* Makefile
 
 install: nbsmtp
-	$(INSTALL) -d $(bindir)
-	$(INSTALL) nbsmtp $(bindir)/nbsmtp
-	$(INSTALL) -d $(mandir)/man8
-	$(INSTALL) nbsmtp.8 $(mandir)/man8/nbsmtp.8
-	$(INSTALL) -d $(mandir)/man5
-	$(INSTALL) nbsmtprc.5 $(mandir)/man5/nbsmtprc.5
+	$(INSTALL) -d $(DESTDIR)$(bindir)
+	$(INSTALL) nbsmtp $(DESTDIR)$(bindir)/nbsmtp
+	$(INSTALL) -d $(DESTDIR)$(mandir)/man8
+	$(INSTALL) nbsmtp.8 $(DESTDIR)$(mandir)/man8/nbsmtp.8
+	$(INSTALL) -d $(DESTDIR)$(mandir)/man5
+	$(INSTALL) nbsmtprc.5 $(DESTDIR)$(mandir)/man5/nbsmtprc.5
 
 doc:
 	@( [ ! -z $(doxygen) ] && $(doxygen) Doxyfile ) || echo "Doxygen wasn't found"
