--- Makefile.in.orig	2005-03-14 00:08:21.000000000 +0100
+++ Makefile.in	2005-05-16 19:40:07.000000000 +0200
@@ -5,7 +5,7 @@
 libexecdir=@libexecdir@
 doxygen=@doxygen@
 bindir=$(prefix)/bin
-mandir=$(prefix)/man
+mandir=@mandir@
 iconpath=$(prefix)/nbsmtp-@PACKAGE_VERSION@
 INSTALL=@INSTALL@
 
@@ -31,15 +31,15 @@
 install: nbsmtp
 	[ "x@OSXMODULE@" = "xyes" ] \
 	&& ( \
-		$(INSTALL) -d $(iconpath); \
-		$(INSTALL) nbsmtp-growl.png $(iconpath)/nbsmtp-growl.png; ) \
+		$(INSTALL) -d $(DESTDIR)$(iconpath); \
+		$(INSTALL) nbsmtp-growl.png $(DESTDIR)$(iconpath)/nbsmtp-growl.png; ) \
 	|| ( true )
-	$(INSTALL) -d $(bindir)
-	$(INSTALL) nbsmtp $(bindir)/nbsmtp
-	$(INSTALL) -d $(mandir)/man8
-	$(INSTALL) nbsmtp.8 $(mandir)/man8/nbsmtp.8
-	$(INSTALL) -d $(mandir)/man5
-	$(INSTALL) nbsmtprc.5 $(mandir)/man5/nbsmtprc.5
+	$(INSTALL) -d $(DESTDIR)$(bindir)
+	$(INSTALL) nbsmtp $(DESTDIR)$(bindir)/nbsmtp
+	$(INSTALL) -d $(DESTDIR)$(mandir)/man8
+	$(INSTALL) nbsmtp.8 $(DESTDIR)$(mandir)/man8/nbsmtp.8
+	$(INSTALL) -d $(DESTDIR)$(mandir)/man5
+	$(INSTALL) nbsmtprc.5 $(DESTDIR)$(mandir)/man5/nbsmtprc.5
 
 doc:
 	@( [ ! -z $(doxygen) ] && $(doxygen) Doxyfile ) || echo "Doxygen wasn't found"
